addon/components/polaris-range-slider.js

```diff
@@ -0,0 +1,289 @@
+import Component from '@ember/component';
+import { guidFor } from '@ember/object/internals';
+import { computed } from '@ember/object';
+import { dasherize, htmlSafe } from '@ember/string';
+import { assign } from '@ember/polyfills';
+import { isNone } from '@ember/utils';
+import { errorId, helpTextId } from '@smile-io/ember-polaris/utils/id';
+import layout from '../templates/components/polaris-range-slider';
+
+function invertNumber(number) {
+  if (Math.sign(number) === 1) {
+    return -Math.abs(number);
+  } else if (Math.sign(number) === -1) {
+    return Math.abs(number);
+  } else {
+    return 0;
+  }
+}
+
+/**
+ * Polaris range slider component.
+ * See https://polaris.shopify.com/components/forms/range-slider
+ */
+export default Component.extend({
+  tagName: '',
+
+  layout,
+
+  /**
+   * Label for the range input
+   *
+   * @property {label}
+   * @type {String}
+   * @public
+   */
+  label: null,
+
+  /**
+   * Adds an action to the label
+   *
+   * @property {labelAction}
+   * @type {Object}
+   * @public
+   */
+  labelAction: null,
+
+  /**
+   * Visually hide the label
+   *
+   * @property {labelHidden}
+   * @type {Boolean}
+   * @default false
+   * @public
+   */
+  labelHidden: false,
+
+  /**
+   * ID for range input
+   *
+   * Defaults to Ember's internal GUID for the component instance
+   *
+   * @property {id}
+   * @type {String}
+   * @public
+   */
+  id: computed(function() {
+    return guidFor(this);
+  }),
+
+  /**
+   * Initial value for range input
+   *
+   * @property {value}
+   * @type {Number}
+   * @default 0
+   * @public
+   */
+  value: 0,
+
+  /**
+   * Minimum possible value for range input
+   *
+   * @property {min}
+   * @type {Number}
+   * @default 0
+   * @public
+   */
+  min: 0,
+
+  /**
+   * Maximum possible value for range input
+   *
+   * @property {max}
+   * @type {Number}
+   * @default 100
+   * @public
+   */
+  max: 100,
+
+  /**
+   * Increment value for range input changes
+   *
+   * @property {step}
+   * @type {Number?}
```

Comment:
typo here `?`

Reply:
I actually did that deliberately to try to indicate that `step` can also be null... happy to remove if you prefer?